system-tests/test/protocol_spec.js

```diff
@@ -50,7 +49,31 @@ describe('capture-protocol', () => {
       }).then(() => {
         const protocolEvents = fs.readFileSync(getFilePath('e9e81b5e-cc58-4026-b2ff-8ae3161435a6.db'), 'utf8')
 
-        systemTests.snapshot('protocol events', normalizeEvents(protocolEvents))
+        systemTests.snapshot('e2e events', normalizeEvents(protocolEvents))
+
+        fs.removeSync(getFilePath('e9e81b5e-cc58-4026-b2ff-8ae3161435a6.db'))
+      })
+    })
+  })
+
+  describe('component', () => {
+    it('verifies the protocol events are correct', function () {
```

Comment:
should we have two tests here, one for `experimentalSingleTabRunMode` on and one with it off?

Reply:
Good idea, updated to also test with the experiment off. [73fef85](https://github.com/cypress-io/cypress/pull/27659/commits/73fef8513d89e7d92f15a74d51467f8ad42e086c)